src/hooks/useError.js

```diff
@@ -0,0 +1,8 @@
+import { useSelector } from 'react-redux';
+
+const useError = action => useSelector(({ actionStatus }) => {
```

Comment:
I can get the error already using `useStatus`, right? maybe it's redundant

Reply:
You can, but it's a bit more verbose, i left the `useLoading` and `useError` implemented "specializations" of the `useStatus`, since many times you just want to know if something is actually loading, or display an error if present. The `useStatus` definitely covers both cases, but takes a bit more code to do so.